AK/Format.cpp

```diff
@@ -688,6 +688,19 @@ void Formatter<char>::format(FormatBuilder& builder, char value)
         return formatter.format(builder, { &value, 1 });
     }
 }
+void Formatter<wchar_t>::format(FormatBuilder& builder, wchar_t value)
+{
+    if (m_mode == Mode::Binary || m_mode == Mode::BinaryUppercase || m_mode == Mode::Decimal || m_mode == Mode::Octal || m_mode == Mode::Hexadecimal || m_mode == Mode::HexadecimalUppercase) {
+        Formatter<unsigned long> formatter { *this };
```

Comment:
This and the following line would read nicer with u64.

Reply:
I now realized that I somehow mixed up the size of `wchar_t` with the `ul` that was required for some of the return values in the tests. I chose `unsigned long` because I wanted a type that would be 4 bytes on x86 and 8 bytes on x86_64, but after testing it seems like `wchar_t` is always 4 bytes in size. I'll amend the commit appropriately to use `u32`.